### PR TITLE
Update Windows compatibility to WSL in tutorials

### PR DESCRIPTION
**Pull request by ygoumaz**
ditto.

This PR should only be merged and synchronized with `ros2/ros2_documentation` once Webots R2023a is released, because WSL changes will only be available when merging `develop` branch to `master` branch in `webots_ros2`.

### CODE REVIEW

File: source/Tutorials/Advanced/Simulators/Webots.rst [hunk 1]
```diff
@@ -219,23 +212,37 @@ The node will be able to communicate with the simulated robot by using a custom
 In your case, you need to run a single instance of this node, because you have a single robot in the simulation.
 But if you had more robots in the simulation, you would have to run one instance of this node per robot.
 ``WEBOTS_CONTROLLER_URL`` is used to define the name of the robot the driver should connect to.
+On Windows, the communication between Webots and the controllers, running in WSL, is possibly via TCP connection.
```
Comment (omichel) [inline source/Tutorials/Advanced/Simulators/Webots.rst]:
```suggestion
On Windows, the communication between Webots (running in Windows) and the controllers (running in WSL) goes through a TCP connection.
```

File: source/Tutorials/Advanced/Simulators/Webots.rst [hunk 2]
```diff
@@ -35,32 +35,16 @@ Prerequisites
...
+This `page <https://github.com/cyberbotics/webots_ros2/wiki/Build-and-Install#windows>`_ explains how to setup such installation. 
 
-.. tabs::
-
-   .. group-tab:: Linux
-
-      .. code-block:: console
+.. code-block:: console
 
+        # On Windows, execute these in the WSL environment
```
Comment (omichel) [inline source/Tutorials/Advanced/Simulators/Webots.rst]: I find this sentence (repeated several times) is a bit confusing as it seems we should execute these lines this ONLY on Windows, not on Linux. I would remove these sentences and add a sentence earlier that explain that all the Ubuntu and ROS commands should be entered in WSL for Windows users.

**Review by omichel — CHANGES_REQUESTED** [review]

File: source/Tutorials/Advanced/Simulators/Webots.rst [hunk 3]
```diff
@@ -35,32 +35,15 @@ Prerequisites
 
 It is recommended to understand basic ROS principles covered in the beginner :doc:`../../../Tutorials`.
 In particular, :doc:`../../Beginner-CLI-Tools/Introducing-Turtlesim/Introducing-Turtlesim`, :doc:`../../Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics`, :doc:`../../Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace`, :doc:`../../Beginner-Client-Libraries/Creating-Your-First-ROS2-Package` and :doc:`../../Intermediate/Launch/Creating-Launch-Files` are useful prerequisites.
-Finally, you will need to install ``webots_ros2_driver`` from a terminal with this command:
+Finally, you will need to install ``webots_ros2_driver`` from a terminal with the following commands. 
+On Windows, a WSL environment must be configured and used to run all the console and ROS commands contained in this tutorial.
```
Comment (omichel) [inline source/Tutorials/Advanced/Simulators/Webots.rst]:
```suggestion
On Windows, a WSL (Windows Subsystem for Linux) environment must be configured and used to run all the Linux and ROS commands contained in this tutorial. 
```

**Review by omichel — CHANGES_REQUESTED** [review]

**Review by omichel — APPROVED** [review]
Thank you.

**Comment by ygoumaz** [comment]
I convert this PR back to draft, as we should only merge it once Webots R2023a is released.

**Comment by ygoumaz** [comment]
Finally, we decided to release the WSL changes earlier in `webots_ros2 2022.1.2-2`. This PR can be merged.

**Pull request merged**